ring-core/test/ring/middleware/test/file.clj

```diff
@@ -93,3 +93,45 @@
       (is (= 200 status))
       (is (= (into #{} (keys headers)) #{"Content-Length" "Last-Modified"}))
       (is (= foo-html body)))))
+
+(deftest test-wrap-file-with-prefer-handler-option
+  (let [processable-file-uri (str "/" (.getName (File. ^String public-dir "bars.txt")))
+        handler (fn handler
+                  ([request]
+                   (if (= (:uri request) processable-file-uri)
+                     {:status 200
+                      :body   "I'm a blog"}
+                     {:status 404 :body "don't know how to process this file"}))
+
+                  ([request respond raise] ;;async handler
+                   (respond (handler request))))
+        app-prefer-handler (wrap-file handler (File. public-dir) {:prefer-handler? true})]
+
+    (testing "middleware serves file"
+      (testing "synchronous"
+        (let [{:keys [status body]} (app-prefer-handler {:request-method :get :uri "/index.html"})]
+          (is (= 200 status))
+          (is (= index-html body))))
+
+      (testing "asynchronous"
+        (let [response (promise)
+              error (promise)]
```

Comment:
Please align let clauses.

Reply:
I'm not sure which alignment you're talking about. I aligned 119 and 120. if that is not what you're referring to. please feel free to align it as you see fit.

Reply:
I mean the clauses should line up. So like:

```
(let [response (promise)
      error    (promise)]
```

It's a lot easier to read when the let clauses are in columns (as long as the lengths aren't too different).